master/internal/db/postgres_checkpoints.go

```diff
@@ -35,6 +35,25 @@ func (db *PgDB) CheckpointByUUIDs(ckptUUIDs []uuid.UUID) ([]model.Checkpoint, er
 	return checkpoints, nil
 }
 
+// GetModelIDsAssociatedWithCheckpoint returns the model ids associated with a checkpoint,
+// returning nil if error.
+func (db *PgDB) GetModelIDsAssociatedWithCheckpoint(ckptUUID uuid.UUID) ([]int32, error) {
```

Comment:
can we have this go through bun `NewRaw` instead of expanding the `PgDB`? maybe it could go into `internal/model` if that works for import cycles

Reply:
added it to pkg/model because i'm not allowed to import internal/model to integration tests.